templates/ceph/managed.html

```diff
@@ -0,0 +1,256 @@
+    <p class="u-sv3">Deploy a fully managed Ceph cluster in a few weeks with these simple steps.</p>
+  </div>
+  <div class="u-fixed-width">
+    <ol class="p-stepped-list">
+      <li class="p-stepped-list__item">
+        <div class="row u-equal-height">
+          <div class="col-6">
+            <h3 class="p-stepped-list__title"> Design </h3>
+						<p>Our experts work with your team to <a href="/ceph/consulting" class=>design a Ceph cluster</a> suited to your needs. We guide you through hardware and configuration decisions, to ensure that the resulting cluster is tailored to meet your performance, capacity, security and integration requirements.</p>
+          </div>
+          <div class="col-6 u-vertically-center u-align--center u-hide--small">
+						{{ image (
+							url="https://assets.ubuntu.com/v1/f5d56df2-OpenStack-consulting+design.svg",
+							alt="Ceph managed design",
+							width="233",
+							height="150",
+							hi_def=True,
+							loading="lazy",
+							attrs={"style": "width: 134px; height: auto"}
+							) | safe
+						}}
+          </div>
+        </div>
+      </li>
+      <li class="p-stepped-list__item">
+        <div class="row u-equal-height">
+          <div class="col-6">
+            <h3 class="p-stepped-list__title"> Model </h3>
+						<p>On completion of the design, all decisions are translated into a reusable model. This model allows for reuse across multiple environments, and guarantees repeatable deployments and integrations with infrastructure-as-code (IaC) and continuous integration and continuous delivery (CI/CD) systems.</p>
+          </div>
+          <div class="col-6 u-vertically-center u-align--center u-hide--small">
+						{{ image (
+							url="https://assets.ubuntu.com/v1/46439b10-2.+model_AW.svg",
+							alt="Ceph managed model",
+							width="228",
+							height="150",
+							hi_def=True,
+							loading="lazy",
+							attrs={"style": "width: 134px; height: auto"}
+							) | safe
+						}}
+          </div>
+        </div>
+      </li>
+      <li class="p-stepped-list__item">
+        <div class="row u-equal-height">
+          <div class="col-6">
+            <h3 class="p-stepped-list__title"> Deploy </h3>
+						<p>Our experts will deploy the Ceph cluster in any location, that could be your own on-premise data center, or a co-location facility. Once your hardware is stood up, we test it end-to-end, ensuring all components are functional, and the networking is correctly configured.  Then we build the cluster, validate it, and finally benchmark it's performance before your data is onboarded. </p>
+          </div>
+          <div class="col-6 u-vertically-center u-align--center u-hide--small">
+						{{ image (
+							url="https://assets.ubuntu.com/v1/0cfe6c2d-build.svg",
+							alt="Ceph managed deploy",
+							width="228",
+							height="150",
+							hi_def=True,
+							loading="lazy",
+							attrs={"style": "width: 134px; height: auto"}
```

Comment:
Sorry to be a pain but could you calculate the width and height to add above and remove the attrs  - thank you!